src-self-hosted/stage2.zig

```diff
@@ -1289,6 +1289,8 @@ pub const ClangArgIterator = extern struct {
         no_exceptions,
         rtti,
         no_rtti,
+        for_linker,
+        linker_input_z,
```

Comment:
Note: named it this way to differentiate it from the mysterious undocumented (?) capital Z clang option.